mir-module/src/main/resources/xsl/metadata/mir-admindata-box.xsl

```diff
@@ -126,4 +128,44 @@
     <xsl:apply-imports/>
   </xsl:template>
 
+  <xsl:template name="print-user-info">
+    <xsl:param name="user"/>
+    <xsl:param name="label"/>
+
+    <xsl:variable name="display-name">
+      <xsl:choose>
+        <xsl:when test="$user/user/realName">
+          <xsl:value-of select="$user/user/realName"/>
+        </xsl:when>
+        <xsl:otherwise>
+          <xsl:value-of select="$user/user/@name"/>
+        </xsl:otherwise>
+      </xsl:choose>
+    </xsl:variable>
+
+    <tr>
+      <td class="metaname">
+        <xsl:value-of select="concat($label, ':')"/>
+      </td>
+
+      <td class="metavalue">
+        <xsl:if test="$user/user/realName">
+          <xsl:attribute name="title">
+            <xsl:value-of select="concat($user/user/@name, '@', $user/user/@realm)"/>
+          </xsl:attribute>
+        </xsl:if>
+
+        <xsl:choose>
+          <xsl:when test="mcrxsl:isCurrentUserInRole('admin')">
```

Comment:
I think it would be better to check if the current user ha the `administrate-users` permission.

For example, a repository operator may also have permitted editors to manage users.